cpptod.dd

```diff
@@ -913,6 +915,130 @@ SecondVariant.method();
 ------
 
 
+$(HR)
+
+$(H3 $(LNAME2 functioninputparameters, Function Input Parameters))
+
+$(H4 The C++ Way)
+
+        $(B C++ does not support explicit input parameters.) If you use a
+        $(I pass-by-value) parameter $(D void function(int inputOnlyImplicitly)) it can just
+        be an input parameter implicitly. If you use a $(I pass-by-reference) parameter
+        usually $(D const) is applied to the parameter
+        $(D void foo(const int& inputOnlyParameterImplicitly)) that a forbidden
+        modification of the parameter in the function leads to a compilation error.
+
+$(CPPCODE
+void foo(int implicitInputParameterPassedByValue)
+{
+    // some code dependent on implicitInputParameterPassedByValue
+    ...
+}
+
+void foo(int& implicitInputParameterPassedByReference)
+{
+    // some code dependent on implicitInputParameterPassedByReference
+    ...
+}
+)
+
+The functions are used as follows:
+
+$(CPPCODE
+int inputArgumentPassedByValue = 42;
+foo(inputArgumentPassedByValue);
+// inputArgumentPassedByValue = 42
+
+int inputArgumentPassedByReference = 42;
+foo(inputArgumentPassedByReference);
+// inputArgumentPassedByReference = 42
+)
+
+$(H4 The D Way)
+
+        D supports explicit input parameters.
```

Comment:
`.` is used before examples above and below, I think it would be the best to the same here for the sake of consistency.

Reply:
Sorry, I didn't understand. Can you elaborate?

Reply:
Have a look at the examples above and below - they use a colon (`:`)